config/smart-proxy.yaml

```diff
@@ -0,0 +1,133 @@
+# foreman-installer main configuration file
+# note current configuration is written to foreman-installer.yaml every time foreman-install is run
+
+## Installer configuration
+# Path to answer file
+:answer_file: /etc/foreman/installer-scenarios.d/smart-proxy-answers.yaml
+:installer_dir: /usr/share/foreman-installer
+# Uncomment if you want to load puppet modules from a specific path, $pwd/modules is used by default
+:modules_dir: /usr/share/foreman-installer/modules
```

Comment:
`modules_dir` was renamed to `module_dirs` and can be a list. The old version is still supported but deprecated.